test_pull_requests

```diff
@@ -359,6 +361,78 @@ def compose_bot_comment(prefix, comment_data={})
   updated_comment
 end
 
+class FlushableStore
+  def initialize(c_file=nil)
```

Comment:
If `c_file` has to exist for this method to do anything, why default it to `nil`?